static/src/stylesheets/_vars.scss

```diff
@@ -23,8 +23,7 @@ $guss-rem-baseline: 16px;
 $inArticleInlineImgWidth: 114px;
 $headerHeight: 36px;
 
-@import 'components/guss-colours/_colours';
-@import 'components/pasteup-palette/src/_palette';
+@import 'pasteup/palette/src';
```

Comment:
thats an interesting point - we may still jiggle things around once this is all localised to front-end. at the moment, this is mainly to duplicate the current behaviour, and tidy up any of the particularly major hoops we've been jumping through (colour, guss-colour etc)